### ISSUE TITLE
CtClass.detach() generating NullPointerException

### ISSUE THREAD

**Issue by SACHDEVHITESH** [jboss-javassist/javassist#442]
This issue is causing the app to not be build

Caused by: java.lang.NullPointerException
    at javassist.ClassPool.cacheCtClass(ClassPool.java:236)
    at javassist.CtClass.detach(CtClass.java:1427)
    at com.microsoft.intune.mam.TransformationUnit.writeClassesToDirectory(TransformationUnit.java:275)
    at com.microsoft.intune.mam.TransformationUnit.writeOutput(TransformationUnit.java:126)
    at com.microsoft.intune.mam.BuildTimeMamifier.mamify(BuildTimeMamifier.java:131)
    at com.microsoft.intune.mam.MamifyTransformBase.transform(MamifyTransformBase.java:195)
    at com.android.build.gradle.internal.pipeline.TransformTask$2.call(TransformTask.java:284)
    at com.android.builder.profile.ThreadRecorder.record(ThreadRecorder.java:69)
